### DISCUSSION TITLE
Importing the locally deployed accounts into wallet.

### DISCUSSION THREAD

**Discussion by sandeepV2** [NomicFoundation/hardhat#5207] [General]
Accounts that are created in local system using `npx hardhat node` .  Can we import these accounts into wallet using a private key for developing a dapp. ?

>  ➜  hardhat-boilerplate git:(master) npx hardhat node
> Started HTTP and WebSocket JSON-RPC server at http://127.0.0.1:8545/
> 
> Accounts
> ========
> 
> WARNING: These accounts, and their private keys, are publicly known.
> Any funds sent to them on Mainnet or any other live network WILL BE LOST.
> 
> Account #0: 0xf39Fd6e51aad88F6F4ce6aB8827279cffFb92266 (10000 ETH)
> Private Key: 0xac0974bec39a17e36ba4a6b4d238ff944bacb478cbed5efcae784d7bf4f2ff80

I tried importing account using the private key provided above, but i did not contain required test ethers. 

<img width="337" alt="Screenshot 2024-05-14 at 9 11 10 AM" src="https://github.com/NomicFoundation/hardhat/assets/52043035/69540444-0bfd-45fa-8f51-92c730cd43e5">

Can you please shares the steps do so, Thank you in advance.

**Comment by kanej** [comment]
The imported account 0 should show ether if your metamask is connected to the local hardhat node as the network.
You have to be very careful that you are not operating on mainnet with those accounts as the private keys are known by all.

**Comment by sandeepV2** [comment]
Thank you @kanej, 
>if your metamask is connected to the local hardhat

Missed on adding the local network to metamask. Added it on different setup, was expecting it to work here. 
Devil is in the details. 

For any one who is hitting this issue, find a article to leverage local network and adding to metamask. 
https://medium.com/@kaishinaw/connecting-metamask-with-a-local-hardhat-network-7d8cea604dc6
